src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/DatasetsTab.java

```diff
@@ -27,7 +25,7 @@ public DatasetsTab(Tab tab, PortalGinInjector ginInjector) {
 		super(tab, ginInjector);
 		this.tab = tab;
 		this.ginInjector = ginInjector;
-		tab.configure(DisplayConstants.DATASETS, "table", DATASETS_HELP, "", EntityArea.DATASETS);
+		tab.configure(DisplayConstants.DATASETS, "dataset", DATASETS_HELP, "", EntityArea.DATASETS);
```

Comment:
Custom icon for datasets tab:
<img width="805" alt="image" src="https://user-images.githubusercontent.com/17580037/144897481-60a10a5c-7029-44e1-8d52-e872e4a2381f.png">